### PR TITLE
Updates per Release30Apr2021

### PR DESCRIPTION
**Pull request by marcfishelquantil**
Burton, kindly review and reply with feedback or approve. Thank you.

### CODE REVIEW

File: docs/portal/edge-configurations/managing-properties.md [hunk 1]
```diff
@@ -32,4 +32,6 @@ Each property has a vertical ellipsis in the **Actions** column. Clicking the el
 - **Undeploy from Production**: [Undeploy](</docs/portal/edge-configurations/deploying-property.md>) the selected property from production.
 - **Delete**: [Delete](</docs/portal/edge-configurations/deleting-property.md>) the selected property.
 - **Deployment History**: [View](</docs/portal/edge-configurations/deploying-property.md>) the deployment history of the selected property.
+- **Clone**: Copies settings into the [Create a Property form](</docs/portal/edge-configurations/creating-property.md>) to create a new property.
+- **Report**: Displays the [Reports page](</docs/portal/reports.md>) for running a Traffic Bandwidth report that can be filtered with the hostnames in the deployed property version. This action is available only for properties that have been deployed to production.
```
Comment (burtonquantil) [inline docs/portal/edge-configurations/managing-properties.md:36]: Based on the current implementation, the user can pick any report type, not just the Traffic Bandwidth one.
I recommend showing more generic text such as:

> Create a report using the hostnames of the deployed property.

File: docs/portal/traffic-management/overview.md [hunk 2]
```diff
@@ -18,4 +18,5 @@ In the left pane, click **Traffic Management** to see a page with a list of the
 Each edge hostname has a vertical ellipsis in the **Actions** column. Clicking the ellipsis shows the available actions that can be performed on the selected edge hostname.
 - **Edit**: [Update](</docs/portal/traffic-management/editing-edge-hostname.md>) the selected edge hostname.
 - **Delete**: [Delete](</docs/portal/traffic-management/deleting-edge-hostname.md>) the selected edge hostname.
-- **Clone**: Copies settings into the Create an Edge Hostname form to create a new edge hostname.
+- **Clone**: Copies settings into the Create an Edge Hostname form to [create a new edge hostname](</docs/portal/traffic-management/creating-edge-hostname.md>).
+- **Report**: Displays the [Reports page](</docs/portal/reports.md>) for running an Edge Hostname Request report that can be filtered with the selected edge host name.
```
Comment (burtonquantil) [inline docs/portal/traffic-management/overview.md:22]: 1) I suggest replacing "that can be filtered with"  with  "on".
2) "host name " should be "hostname".

File: docs/portal/edge-configurations/managing-properties.md [hunk 3]
```diff
@@ -32,4 +32,6 @@ Each property has a vertical ellipsis in the **Actions** column. Clicking the el
 - **Undeploy from Production**: [Undeploy](</docs/portal/edge-configurations/deploying-property.md>) the selected property from production.
 - **Delete**: [Delete](</docs/portal/edge-configurations/deleting-property.md>) the selected property.
 - **Deployment History**: [View](</docs/portal/edge-configurations/deploying-property.md>) the deployment history of the selected property.
+- **Clone**: Copies settings into the [Create a Property form](</docs/portal/edge-configurations/creating-property.md>) to create a new property.
```
Comment (burtonquantil) [inline docs/portal/edge-configurations/managing-properties.md:35]: Are the words "to create a new property" necessary? They seem redundant to me given the reference to the form.

File: docs/portal/traffic-management/overview.md [hunk 4]
```diff
@@ -18,4 +18,5 @@ In the left pane, click **Traffic Management** to see a page with a list of the
 Each edge hostname has a vertical ellipsis in the **Actions** column. Clicking the ellipsis shows the available actions that can be performed on the selected edge hostname.
 - **Edit**: [Update](</docs/portal/traffic-management/editing-edge-hostname.md>) the selected edge hostname.
 - **Delete**: [Delete](</docs/portal/traffic-management/deleting-edge-hostname.md>) the selected edge hostname.
-- **Clone**: Copies settings into the Create an Edge Hostname form to create a new edge hostname.
+- **Clone**: Copies settings into the Create an Edge Hostname form to [create a new edge hostname](</docs/portal/traffic-management/creating-edge-hostname.md>).
```
Comment (burtonquantil) [inline docs/portal/traffic-management/overview.md:21]: Are the words "to create a new edge hostname" really necessary?  They seem redundant to me given the reference to the form.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Please check my feedback. 
Some of it is arguable one way or the other, but there is definitely one error that must be corrected.  Use "hostname".

File: docs/portal/edge-configurations/managing-properties.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/edge-configurations/managing-properties.md:36]: Good comment. Instead of "Create," I would prefer to use "Run."

Comment (burtonquantil) [reply docs/portal/edge-configurations/managing-properties.md:36]: @marcfishelquantil  OK

File: docs/portal/traffic-management/overview.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/portal/traffic-management/overview.md:22]: Both corrections made.

File: docs/portal/traffic-management/overview.md (hunk 4 above)
Comment (marcfishelquantil) [reply docs/portal/traffic-management/overview.md:21]: Could be construed as redundant. So I deleted "new" from this topic and also from Managing Properties.

File: docs/portal/edge-configurations/managing-properties.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/portal/edge-configurations/managing-properties.md:35]: While I generally agree that less is more, these words add important context to the meaning being conveyed to users.

**Review by burtonquantil — APPROVED** [review]
OK to merge.

**Pull request merged**